internal/cmd/perfcomp/main.go

```diff
@@ -0,0 +1,357 @@
+// Copyright (C) MongoDB, Inc. 2025-present.
+//
+// Licensed under the Apache License, Version 2.0 (the "License"); you may
+// not use this file except in compliance with the License. You may obtain
+// a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+
+// This module cannot be included in the workspace since it requires a version of Gonum that is not compatible with the Go Driver.
+// Must use GOWORK=off to run this test.
+
+package main
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"math"
+	"os"
+	"strings"
+	"time"
+
+	"go.mongodb.org/mongo-driver/v2/bson"
+	"go.mongodb.org/mongo-driver/v2/mongo"
+	"go.mongodb.org/mongo-driver/v2/mongo/options"
+	"gonum.org/v1/gonum/mat"
+)
+
+type OverrideInfo struct {
+	OverrideMainline bool        `bson:"override_mainline"`
+	BaseOrder        interface{} `bson:"base_order"`
```

Comment:
Optional: You can use [any](https://pkg.go.dev/builtin#any) instead of `interface{}`, which is semantically identical but slightly cleaner looking syntax. This applies to any use of `interface{}`.